### ISSUE TITLE
Laundry building fragment illegalstateexception

### ISSUE THREAD

**Issue by jasontsh** [pennlabs/penn-mobile-android#346]
`Fatal Exception: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
       at android.support.v4.app.FragmentManagerImpl.checkStateLoss(FragmentManager.java:1500)
       at android.support.v4.app.FragmentManagerImpl.enqueueAction(FragmentManager.java:1518)
       at android.support.v4.app.BackStackRecord.commitInternal(BackStackRecord.java:634)
       at android.support.v4.app.BackStackRecord.commit(BackStackRecord.java:613)
       at com.pennapps.labs.pennmobile.LaundryBuildingFragment.toLaundryMachine(LaundryBuildingFragment.java:73)
       at com.pennapps.labs.pennmobile.LaundryBuildingFragment.onListItemClick(LaundryBuildingFragment.java:58)
       at android.support.v4.app.ListFragment$2.onItemClick(ListFragment.java:58)
       at android.widget.AdapterView.performItemClick(AdapterView.java:339)`

**Comment by jasontsh** [comment]
`Fatal Exception: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
       at android.support.v4.app.FragmentManagerImpl.checkStateLoss(FragmentManager.java:1500)
       at android.support.v4.app.FragmentManagerImpl.enqueueAction(FragmentManager.java:1518)
       at android.support.v4.app.BackStackRecord.commitInternal(BackStackRecord.java:634)
       at android.support.v4.app.BackStackRecord.commit(BackStackRecord.java:613)
       at com.pennapps.labs.pennmobile.DiningFragment.onListItemClick(DiningFragment.java:89)
       at android.support.v4.app.ListFragment$2.onItemClick(ListFragment.java:58)`

Basically the same thing, but in dining fragment
